_episodes/33.md

```diff
@@ -0,0 +1,266 @@
+* Support for groups in OAuth2 claims.
+* Column comments in ClickHouse connector.
+* Write Bloom filters in ORC files.
+* Procedure for optimizing Iceberg tables.
+
+Trino 370
+
+* Add CLI support for ARM64
+* Improved performance for ORC
+* Improved performance for map and row types
+* Reduced latency for OAuth2.0 authentication
+
+Trino 371
+
+* Support for secrets and user group selector in resource group manager.
+* Support AWS role session name in S3 security mapping configuration.
+* Many bug fixes.
+
+Notes from Manfred
+
+* Add support for using PostgreSQL and Oracle as backend database for resource
+  groups.
+* Remove `spill-order-by`,  `spill-window-operator`, and
+  `query.max-total-memory-per-node`.
+* Add support for `ALTER MATERIALIZED VIEW ... SET PROPERTIES` in the engine.
+* Prevent hanging query execution on failures with phased execution policy.
+* Support for renaming schemas in PostgreSQL and Redshift connectors.
+* Lots of improvements on Clickhouse connector, thanks Yuya!
+* Update to newer ClickHouse version removed support for Altinity 20.3
+* `$properties` table and other hidden tables in Iceberg connector, including
+  docs.
+* Automatically adjust `ulimit` setting when using the RPM package.
+* Docker images changes to UBI.
+* Remove support/need for `allow-drop-table` catalog property in JDBC connectors.
+* A bunch of SPI changes
+* DML with Iceberg connector with fault tolerant mode and more Tardigrade improvements
+* Drop support for Kudu 1.13.0
+  
+More detailed information is available in the [Trino
+369](https://trino.io/docs/current/release/release-369.html), [Trino
+370](https://trino.io/docs/current/release/release-370.html), and [Trino
+371](https://trino.io/docs/current/release/release-371.html) release notes.
+
+## Concept of the month: High Availability with Trino
+
+Goldman Sachs uses Trino to reduce last-mile ETL and provide a unified way of 
+accessing data through federated joins. Making a variety of data sets from 
+different sources available in one spot for our data science team was a tall 
+order. Data must be quickly accessible to data consumers and systems like Trino
+must be reliable for users to trust this singular access point for their data.
+
+In order for analysts and data scientists to use these services, they first need
+to trust in the system. It was vital to Goldman Sachs that Trino have High
+Availability so that in the event of failure, there was always another Trino
+cluster available to take requests.
+
+### Integrating Trino into the Goldman Sachs internal ecosystem 
+
+Before high availability was a concern, the team had to first integrate Trino to
+meet their requirements. This included integrating with internal security 
+systems, observability systems, and credential stores. It also meant
+adding integration with their governance services that manage cataloguing
+services and data discovery engines. Finally, while many of the connectors that
```

Comment:
Finally, while many of the Trino connectors that the team intended to use exist, there were many ...